conftest.py

```diff
@@ -402,3 +414,52 @@ def pytest_sessionfinish(session, exitstatus):
         junit_modifyreport(
             _JSON_REPORT_FILE(), _XML_REPORT_FILE(), junit_properties=context.scenario.get_junit_properties(),
         )
+
+        export_feature_parity_dashbaord(session)
+
+
+def export_feature_parity_dashbaord(session):
+    data = session.config._json_report.report  # pylint: disable=protected-access
+
+    result = {
+        "runUrl": session.config.option.report_run_url,
+        "runDate": data["created"],
+        "environment": session.config.option.report_environment,
+        "testSource": "systemtests",
+        "language": context.scenario.library.library,
+        "variant": context.scenario.weblog_variant,
+        "testedDependencies": [
+            {"name": name, "version": str(version)} for name, version in context.scenario.components.items()
+        ],
+        "scenario": context.scenario.name,
+        "tests": [convert_test_to_feature_parity_model(test) for test in data["tests"]],
+    }
+
+    with open(f"{context.scenario.host_log_folder}/feature_parity.json", "w", encoding="utf-8") as f:
+        json.dump(result, f, indent=2)
+
+
+def convert_test_to_feature_parity_model(test):
+    result = {
+        "path": test["nodeid"],
+        "lineNumber": test["lineno"],
```

Comment:
are we using the line number as unique identifier? In the case of parametrized tests (parametric, onboarding) it's always same line number.